### ISSUE TITLE
Ctrl + (+) and Ctrl + (-) are missing form Keyboard shortcuts menu

### ISSUE THREAD

**Issue by JulianGro** [zulip/zulip-desktop#1031]
**Describe the bug**
`Ctrl` + `+` and `Ctrl` + `-` are missing form Keyboard shortcuts menu in the bottom right.

**Expected behavior**
It should tell you about these important shortcuts.

**Desktop (please complete the following information):**
 - Operating System:  Linux Mint 19.3 (Ubuntu 18.04.3)
 - Zulip Desktop Version: 5.4.0

**Additional context**
Zulip is pretty small by standard. A lot of users will want to increase its size by pressing `Ctrl` + `+`

**Comment by akashnimare** [comment]
Does this shortcut work for you -

![image](https://user-images.githubusercontent.com/2263909/93981086-1c04f080-fd9d-11ea-92c9-a328c4cf9808.png)

**Comment by JulianGro** [comment]
That is something I have missed. I cannot press `Ctrl` + `=`, as `=` is not a dedicated button on a QWERTZ keyboard.

**Comment by simolev** [comment]
Contrary to what the menu says, zooming in for me is `Ctrl` + `+` and not `Ctrl` + `=`.
Windows 6.1.7601, Zulip Desktop v5.4.3, Italian keyboard layout.

When using the EN-US keyboard layout, `Ctrl` + `=` zooms in, but does so by two steps instead of one; this is both visually noticeable as it happens in two steps, and can easily be checked: pressing `Ctrl` + `-` *twice* is needed to go back to what it was.

**Comment by a-saksham** [comment]
zulipbot claim --force

**Comment by a-saksham** [comment]
@zulipbot claim --force

**Comment by skshamagarwal** [comment]
@akashnimare can you assign this issue to me.

**Comment by skshamagarwal** [comment]
#1063 **Issue fixed - ready for review**

**Comment by timabbott** [comment]
The fix for this issue likely does not involve adding all desktop app shortcuts these to the in-webapp keyboard shortcuts menu; instead, we should probably just add a simple hint in the webapp codebase conditional on `electron_bridge` directing users how to find the desktop app keyboard shortcuts as well.

**Comment by nuke-web3** [comment]
I agree a note on the `?` keyboard shortcuts menu to the effect of "Look at the menu bar for more system-related shortcuts (zoom,dev tools, etc.)"
